doc/api/swagger.json

```diff
@@ -2032,7 +1741,28 @@
                 "emails": {
                   "type": "array",
                   "items": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                      "address": {
+                        "type": "string"
+                      },
+                      "is_primary": {
+                        "type": "boolean"
+                      },
+                      "is_backup": {
```

Comment:
may be I don't understand well : It looks like the recover email is set in contact details of the user.
I am not sure it is good idea:

 - a user can delete a contact details ... the recovery emal as well
 - so we both need to add a constraint on deletion and create the engine to change the recovery email etc. (more code = more bug)
 - it also requires more UX reflexion to manage this kind of cases.
 - a simple contact has no is_backup concept
 - right now we can't store a confirmation or validation of this email

I've an other proposition: add a "custom" field to the user : the recovery_email and the date of the validation; (it might be usefull to add the history of this field).
On user creation we can simply duplicate the recover_email into contact_details